src/utils/queryBuilder.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+// Builds up filter string by filter type
+const eventFilters = new Map([
+  ['startDate', val => `start_date gt ${moment(val).format('YYYY-MM-DD')}`],
```

Comment:
That makes sense to me.  The demo query string I had used `gt` but I'll change it